ndnsd/discovery/service-discovery.cpp

```diff
@@ -343,8 +370,8 @@ ServiceDiscovery::wireDecode(const ndn::Block& wire)
 
```

Comment:
If this wireDecode function just returns the consumerReply it has nothing to do with this class. Make it a free/static function. Why are you saving the m_wire - what do you gain out of it? You are not initializing any other class members...
I guess question is how is this wireDecode used? As far as I can tell it is static.

Reply:
I have made wireDecode static for now. It can be static as long as we find a better use case.